2021-05-31 16:32:45|INFO|gpload session started 2021-05-31 16:32:45
2021-05-31 16:32:45|INFO|setting schema 'public' for table 'texttable2'
2021-05-31 16:32:45|INFO|started gpfdist -p 8081 -P 8082 -f "/home/gpadmin/workspace/gpdb/gpMgmt/bin/gpload_test/gpload2/data_file.txt" -t 30
2021-05-31 16:32:45|INFO|did not find an external table to reuse. creating ext_gpload_reusable_c5db02a4_c1ea_11eb_ad06_0050569ea4ff
2021-05-31 16:32:45|ERROR|ERROR:  missing data for column "s2"  (seg0 slice1 127.0.0.1:6002 pid=15244)
CONTEXT:  External table ext_gpload_reusable_c5db02a4_c1ea_11eb_ad06_0050569ea4ff, line 1 of gpfdist://127.0.0.1:8081//home/gpadmin/workspace/gpdb/gpMgmt/bin/gpload_test/gpload2/data_file.txt: "123456789	abcd"
 encountered while running INSERT INTO public."texttable2" ("s1","s2") SELECT "s1","s2" FROM ext_gpload_reusable_c5db02a4_c1ea_11eb_ad06_0050569ea4ff
2021-05-31 16:32:45|INFO|rows Inserted          = 0
2021-05-31 16:32:45|INFO|rows Updated           = 0
2021-05-31 16:32:45|INFO|data formatting errors = 0
2021-05-31 16:32:45|INFO|gpload failed
